versions.tf

```diff
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 terraform {
-  required_version = ">= 1.0.0"
+  required_version = ">= 1.4.5"
```

Comment:
Just be aware some customers may have earlier versions of terraform and this could break their automation.

Reply:
Yes it will be "breaking" for anyone who does not have Terraform 1.4.5 installed and is not using the Docker image.
Since this PR is marked as "breaking" I'll be sure to include release notes advising users to update their terraform versions before consuming this release.